prefect_databricks/flows.py

```diff
@@ -306,18 +340,76 @@ async def jobs_runs_submit_and_wait_for_completion_flow(notebook_path, **base_pa
                 f"encountered an internal error: {jobs_runs_state_message}.",
             )
         else:
-            logger.info(
-                "Databricks Jobs Runs Submit (%s ID %s) has %s state. "
-                "Waiting for %i seconds.",
-                run_name,
-                multi_task_jobs_runs_id,
-                jobs_runs_life_cycle_state.lower(),
-                poll_frequency_seconds,
-            )
+            logger.info("Waiting for %s seconds.", poll_frequency_seconds)
             await asyncio.sleep(poll_frequency_seconds)
             seconds_waited_for_run_completion += poll_frequency_seconds
 
     raise DatabricksJobRunTimedOut(
         f"Max wait time of {max_wait_seconds} seconds exceeded while waiting "
         f"for job run ({run_name} ID {multi_task_jobs_runs_id})"
     )
+
+
+def __log_state(
+    array: Dict[str, Any],
```

Comment:
The name of this parameter is confusing. It's named array, but it's expected to be a dictionary type. Could you give it a more descriptive name?